### ISSUE TITLE
Figure 1 on 8.4 needs alt-text for screen-readerability

### ISSUE THREAD

**Issue by walkerje** [pearcej/cpp4python#58]
Figure needs alt-text as an accessibility feature. The text should be as succinctly descriptive as possible in regards to what the figure actually shows.
